fearless/Modules/AccountCreate/AccountCreatePresenter.swift

```diff
@@ -268,8 +268,25 @@ extension AccountCreatePresenter: AccountCreatePresenterProtocol {
                 chainId: model.chain.chainId
             )
             wireframe.confirm(from: view, flow: .chain(request))
+        case .backup:
+            let request = MetaAccountImportMnemonicRequest(
+                mnemonic: mnemonic,
+                username: usernameSetup.username,
+                substrateDerivationPath: substrateDerivationPath,
+                ethereumDerivationPath: ethereumDerivationPath,
+                cryptoType: selectedCryptoType
+            )
+            wireframe.showBackupCreatePassword(
+                request: request,
+                from: view
+            )
         }
     }
+
+    func didTapBackupButton() {
+        flow = .backup
```

Comment:
why changed manually ?

Reply:
Flow changed when we tapped with google button